### PR TITLE
Updating host-process images to use the new base image

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Reason for PR**:

Updating host-process images to use the new base image

**Issue Fixed**:

Issue #

**Requirements**

- [ ] Sqaush commits 
- [ ] Documentation
- [ ] Tests

**Notes**:

### CODE REVIEW

File: hostprocess/eventflow-logger/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
-ARG OS=1809
-FROM mcr.microsoft.com/windows/nanoserver:${OS} as runtime
+FROM mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v0.1.0 as runtime
```
Comment (jsturtevant) [inline hostprocess/eventflow-logger/Dockerfile:1]: this image is currently using Docker not buildx so I think it won't work. Will need to update the build scripts

Comment (marosset) [reply hostprocess/eventflow-logger/Dockerfile:1]: Yea, I noticed taht. Same w/ csi-proxy.
I might just update the images that are building w/ buildkit in the PR and then open a separate PR for these 2 images.

**Pull request merged**